src/OrleansGetEventStore/GetEventStoreConfigurationExtensions.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Orleans.Runtime.Configuration
+{
+
+    /// <summary>
+    /// Extension methods for configuration classes specific to OrleansEventSourcing.dll 
+    /// </summary>
+    public static class GetEventStoreConfigurationExtensions
+    {
+        /// <summary>
+        /// Adds a log consistency provider for a GetEventStore
+        /// </summary>
+        /// <param name="config">The cluster configuration object to add provider to.</param>
+        /// <param name="providerName">The provider name.</param>
+        public static void AddGetEventStoreProvider(
+            this ClusterConfiguration config,
+            string providerName = "GetEventStore",
+            string host = "localhost:1113",
+            string username = "admin",
+            string password = "changeit")
```

Comment:
They are optional. These defaults are the standard defaults of GetEventStore, it makes sense to match them.